wpilibcExamples/src/main/cpp/examples/ArmBot/cpp/RobotContainer.cpp

```diff
@@ -21,8 +21,8 @@ RobotContainer::RobotContainer() {
   // Set up default drive command
   m_drive.SetDefaultCommand(frc2::cmd::Run(
       [this] {
-        m_drive.ArcadeDrive(-m_driverController.GetLeftY(),
-                            -m_driverController.GetRightX());
+        m_drive.ArcadeDrive(-m_driverController.GetHID().GetLeftY(),
```

Comment:
This shouldn't be like this. The command HID objects should forward the method calls (like in Java).